### ISSUE TITLE
When I run train.py, it shows "Skipping loading of weights for layer conv2d_67 due to mismatch in shape"

### ISSUE THREAD

**Issue by tonnkie** [qqwweee/keras-yolo3#98]
/usr/local/lib/python3.6/site-packages/keras/engine/saving.py:1009: **UserWarning: Skipping loading of weights for layer** conv2d_59 due to mismatch in shape ((1, 1, 1024, 75) vs (255, 1024, 1, 1)).
  weight_values[i].shape))
/usr/local/lib/python3.6/site-packages/keras/engine/saving.py:1009: UserWarning: Skipping loading of weights for layer conv2d_59 due to mismatch in shape ((75,) vs (255,)).
  weight_values[i].shape))
/usr/local/lib/python3.6/site-packages/keras/engine/saving.py:1009: UserWarning: Skipping loading of weights for layer conv2d_67 due to mismatch in shape ((1, 1, 512, 75) vs (255, 512, 1, 1)).
  weight_values[i].shape))
/usr/local/lib/python3.6/site-packages/keras/engine/saving.py:1009: UserWarning: Skipping loading of weights for layer conv2d_67 due to mismatch in shape ((75,) vs (255,)).
  weight_values[i].shape))
/usr/local/lib/python3.6/site-packages/keras/engine/saving.py:1009: UserWarning: Skipping loading of weights for layer conv2d_75 due to mismatch in shape ((1, 1, 256, 75) vs (255, 256, 1, 1)).
  weight_values[i].shape))
/usr/local/lib/python3.6/site-packages/keras/engine/saving.py:1009: UserWarning: Skipping loading of weights for layer conv2d_75 due to mismatch in shape ((75,) vs (255,)).
  weight_values[i].shape))

 **I am wondering if it matters the whole networks design?**

**Comment by stefanbo92** [comment]
These layers are the three output layers of the yolo network. You probably got the message because you changed the number of classes and therefore the shape of the last conv layers will change aswell. So you can just ignore the message, the network will work just fine.

**Comment by 18814181500** [comment]
Hi, did you have solved the problem yet? I run the train.py with the voc dateset and I met the same issue. what's worse,I got the MemoryError following:
train(model, image_data/255., y_true, log_dir=log_dir)
MemoryError

**Comment by dleam** [comment]
@18814181500 stefanbo92 is right. The pretrained yolo.h5 is trained on coco and the final output is (80 coco classes + 5) *3 =255 according to the paper. The output of VOC is ( 20 voc classes + 5) *3 =75.

**Comment by metromark** [comment]
Im getting the below error when I tried using trained_weights_final.h5 on yolo.py to try the model on a sample image. Im thinking this is due to mismatch as the above issues. I trained it with VOC2007_train.txt 

ValueError: Dimension 0 in both shapes must be equal, but are 1 and 75. Shapes are [1,1,1024,255] and [75,1024,1,1]. for 'Assign_360' (op: 'Assign') with input shapes: [1,1,1024,255], [75,1024,1,1].

I am wondering how I might solve, does any one know how?
